topi/include/topi/transform.h

```diff
@@ -585,7 +585,7 @@ inline Tensor strided_slice(const Tensor& x,
 inline Array<Tensor> split_sections(const Tensor& x,
                                     int num_sections,
                                     int axis,
-                                    std::string name = "tensor",
+                                    std::string name = "T_split",
```

Comment:
I think it would be better to use full function names for consistency, even if they are quite long (i.e. `T_split_sections`).